config.py

```diff
@@ -1,4 +1,4 @@
 PREFIX = '!'
 ADMINS = [698457845301248010] # replace this with your discord ID if hosting your own bot
 LOG_FILE = 'log.txt'
-USERS_FILE = 'users.json'
\ No newline at end of file
+USERS_FILE = 'data.json'
```

Comment:
this variable is still called "USERS_FILE", change this pls

Reply:
done